wildfly-base/src/main/docker/Dockerfile

```diff
@@ -41,10 +41,6 @@ COPY dependencies/jts-${wildfly.hibernate.module.jts.version}.jar $HIBERNATE_MOD
 COPY dependencies/geolatte-geom-${wildfly.hibernate.module.geolatte.version}.jar $HIBERNATE_MODULE
 COPY dependencies/hibernate-spatial-4x-to-5x-wrapper-${project.version}.jar $HIBERNATE_MODULE
 
-ENV YASSON_MODULE $JBOSS_HOME/modules/system/layers/base/org/eclipse/yasson/main
```

Comment:
I think this might cause problems for asset client

Reply:
oops